README.md

```diff
@@ -541,3 +541,100 @@ $ oc new-app --template=manageiq \
   -p APPLICATION_IMG_TAG=latest \
   ...
 ```
+
+## Configuring External Authentication
+Configuring the _httpd_ pod for external authentication is done by updating the _httpd-auth-configs_ configuration map to include all necessary config files and certificates. Upon startup, the _httpd_ pod overlays its files with the ones specified in the _auth-configuration.conf_ file in the configuration map. This is done by the _initialize-httpd-auth_ service that runs before _httpd_.
```

Comment:
Instead of italics can you use backticks, to make it preformatted?  I find it hard to read with italics.